starwars/src/components/Characters.js

```diff
@@ -0,0 +1,47 @@
+import React, {useEffect, useState} from "react";
+import axios from "axios";
+import styled from "styled-components";
+import CharacterCards from "./CharacterCards";
+
+
+const StyledCard = styled.div`
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;    
+    margin: 1%;
+   
+`;
+
+export default function Characters() {
+    const [characters, setCharacters] = useState([]);
+
+    useEffect(() => {
+        axios
+            .get('https://swapi.co/api/people/')
+            .then(response => {
+                console.log(response);
+                setCharacters(response.data.results);
+            })
+            .catch(error => {
+                console.log("The data was not returned", error);
+              });
+    }, []);
+
+    return (
+        <StyledCard>
+        {characters.map((data, index) => {
```

Comment:
Great job using the key property when using multiple elements of the same type. This is good practice. Although, using the id from the data object might be slightly better as it will make key value more unique and reduce amount of code.